### DISCUSSION TITLE
Nextjs getStaticPaths. how to get more than 1000 row when select. only one table

### DISCUSSION THREAD

**Discussion by langbrowser** [supabase/supabase#6469] [Questions]
Using Nextjs with getStaticPaths to get value from table but can only get 1000 row when select. only need for one table so don't want to change Max Rows in API > Setting.

Try with supabase function but call it also only get 1000 row. 

How should I do it. 

example 
```
 const { data, error } = await supabase
        .from ('t_product')
        .select('product_url')';
```

**Comment by GaryAustin1** [comment]
I can think of three options...
1: up the max row setting
2: do multiple requests with pagination using .range
3: have an rpc function convert the rows to a json array in sql and pass that back.
